pstress/mysqld_options_ms_80.txt

```diff
@@ -3158,6 +3822,24 @@
 --slow-launch-time=1125899906842624
 --slow-query-log=1
 --slow-query-log=0
+--slow-query-log-always-write-time=10
```

Comment:
Add this option in exclusion list. We dont want anything to change wrt to writing logs